.gitignore

```diff
@@ -13,7 +14,15 @@ summary.log
 
 # PHPStan
 .php_cs.cache
+phpstan.neon
 
 # PHPUnit
+phpunit.xml
 .phpunit.result.cache
 reports
+
+# Infection
+core.*
```

Comment:
I have no idea what infection does, or how it works, but I can nowhere find in it's documentation how/why it created these `core.1234` files in the projection root when I run it. So I just added to the ignore here.